mopidy_listenbrainz/ext.conf

```
@@ -2,3 +2,5 @@
 enabled = true
 token = 
 url = api.listenbrainz.org
+import_playlists = false
+search_schemes = "local:"
```

```suggestion
search_schemes = local:
```

the quoting was breaking url validation